Lib/test/test_subprocess.py

```diff
@@ -844,7 +844,7 @@ def test_one_environment_variable(self):
             cmd = ["CMD", "/c", "SET", "fruit"]
         with subprocess.Popen(cmd, stdout=subprocess.PIPE, env=newenv) as p:
             stdout, _ = p.communicate()
```

Comment:
```suggestion
            stdout, stderr = p.communicate()
            if p.returncode and support.verbose:
                print("STDOUT:", stdout.decode("ascii", "replace"))
                print("STDERR:", stderr.decode("ascii", "replace"))
            self.assertEqual(p.returncode, 0)
```

We may need something like this to see more about the failure to run - last buildbot just showed empty stdout.